mithril-aggregator/src/services/prover.rs

```diff
@@ -160,7 +174,7 @@ mod tests {
             let hash = format!("tx-{i}");
             transactions.push(CardanoTransaction::new(
                 &hash,
-                10 * i as u64,
+                max(0, 10 * i - 1) as u64,
```

Comment:
It's a bit tricky to know what is the block number value for each transaction.
A comment can help to not have to guess.

Reply:
I added the sequence produced by the formula in a comment